CMakeLists.txt

```diff
@@ -179,6 +175,9 @@ set(Opus_PUBLIC_HEADER
     ${CMAKE_CURRENT_SOURCE_DIR}/include/opus_projection.h
     ${CMAKE_CURRENT_SOURCE_DIR}/include/opus_types.h)
 
+add_library(opus ${opus_sources} ${opus_sources_float} ${Opus_PUBLIC_HEADER})
```

Comment:
Remove ${Opus_PUBLIC_HEADER} for now
If you want then you can create another pull request were you add all headers including the public header to the project.

Reply:
The public headers have to be included here or they are not installed into the built framework.